funnel/forms/project.py

```diff
@@ -39,22 +39,17 @@ class ProjectForm(forms.Form):
         validators=[forms.validators.DataRequired()],
         filters=[forms.filters.strip()],
     )
-    location = forms.StringField(
-        __("Location"),
-        validators=[forms.validators.DataRequired(), forms.validators.Length(max=50)],
-        filters=[forms.filters.strip()],
-        description=__("Eg. Bangalore, Mumbai, Pune"),
-    )
     tagline = forms.StringField(
         __("Tagline"),
         validators=[forms.validators.DataRequired(), forms.validators.Length(max=250)],
         filters=[forms.filters.strip()],
         description=__("One line description of the project"),
     )
-    description = forms.MarkdownField(
-        __("Project description"),
-        validators=[forms.validators.DataRequired()],
-        description=__("Landing page contents"),
+    location = forms.StringField(
+        __("Location"),
+        validators=[forms.validators.DataRequired(), forms.validators.Length(max=50)],
+        filters=[forms.filters.strip()],
+        description=__("Eg. Bangalore, Mumbai, Pune, Online, Anywhere etc"),
```

Comment:
Could this be better?

Reply:
Use Hasjob’s hint and validator, but place “Online” first here (no “Anywhere”).